recipes/bandit/all/test_package/conanfile.py

```diff
@@ -0,0 +1,29 @@
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps", "VirtualRunEnv"
```

Comment:
```suggestion
    generators = "CMakeDeps", "VirtualRunEnv"
    test_type = "explicit"
```
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#changes-in-the-test-package-recipe